src/applications/disability-benefits/all-claims/config/781/index.js

```diff
@@ -117,6 +117,13 @@ export function createFormConfig781(iterations) {
         uiSchema: individualsInvolvedFollowUp.uiSchema(index),
         schema: individualsInvolvedFollowUp.schema(index),
       },
+      [`incidentSupport2${index}`]: {
```

Comment:
This just looks like a duplicate of the ```[`incidentSupport${index}`]``` auto-generated pages? Where is the extra content coming from? They both use the same schema & uiSchema.

Reply:
This is a duplicate of [`incidentSupport${index}`]. UX wanted another support screen after user answers questions about individuals who have been killed or injured.

Reply:
Ok, only concern here is that the title will be the same, so it may be difficult for veterans to understand which page they're on and how it's different from what they just filled out.

Reply:
If it's not possible to change the title or description, maybe put a comment directly in the code? I could see us getting confused down the road and not remembering why this is so.

Reply:
@markgreenburg Agreed updated properties to reflect and additional "break" page. The thought process was this could be the most stressful page in the entire flow and needed an additional follow up break page.